README.md

```diff
@@ -9,14 +9,16 @@ Currently, only the ability to upload packages to the repository via HTTP is
 supported. S3 is used as storage.
 
 ## Table of contents
-* [Getting started](#getting-started)
-  * [Prerequisites](#prerequisites)
-  * [Download](#download)
-  * [Install dependencies](#install-dependencies)
-  * [Run](#run)
-  * [Usage](#usage)
-* [Configuration](#configuration)
-* [Caution](#caution)
+- [RWS](#rws)
```

Comment:
I think adding `RWS` (it would just add another level of nesting) and `Table of contents` to `contents` doesn't make sense . And these changes are not related with commit.

Reply:
fixed

Reply:
`Table of contents` hasn't been deleted.

Reply:
It's a problem with my IDE. VS Code has a plugin which makes changes. 
Fixed.